.pre-commit-config.yaml

```diff
@@ -6,17 +6,13 @@ repos:
     rev: 20.8b1
     hooks:
       - id: black
-
   - repo: https://github.com/alessandrojcm/commitlint-pre-commit-hook
     rev: v5.0.0
     hooks:
       - id: commitlint
         additional_dependencies: ['@commitlint/config-conventional']
         stages: [commit-msg]
-
-  - repo: https://github.com/alessandrojcm/commitlint-pre-commit-hook
-    rev: v5.0.0
+  - repo: https://github.com/pycqa/isort
+    rev: 5.8.0
     hooks:
-      - id: commitlint-travis
-        additional_dependencies: ['@commitlint/config-conventional']
-        stages: [manual]
+      - id: isort
```

Comment:
How does this get the required arguments?

Reply:
Oops good point, moving all the options to `pyproject.toml` so it's picked up consistently.